index.html

```diff
@@ -193,20 +194,27 @@ <h2 class="conta">CONTACT US</h2>
     if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
 </p>
 
-<form >
-    <input class="text-inp" type="text" placeholder=" Text Input">
-
-    <input class="email" type="text" placeholder=" Jeremy@gmail.com">
-
-    <textarea class="send-text" cols="30" rows="10" placeholder="
-Enter text here"></textarea>
-</form>
-
-<button type="button" class="btn-sec"> See Project </button>
-
-<div class="cont-line">.</div>
-
-
+    <form action="https://formspree.io/f/mlezygyr" method="post">
+        <ul class="input-list">
+         <li>
+           <label for="name">Name:</label>
+           <input type="text" id="name" name="user_name" placeholder="  Name" maxlength="30" required>
+         </li>
+         <li>
+           <label for="mail">E-mail:</label>
+           <input type="email" id="mail" name="user_email" placeholder="  E-mail" required>
+         </li>
+         <li>
+           <label for="msg">Message:</label>
+           <textarea cols="33" rows="16"  id="msg" name="user_message" 
+           placeholder=" 
+ Enter text here" maxlength="500" required ></textarea>
+         </li>
+         <li class="button">
+           <button type="submit" id="form-btn"> see project </button>
```

Comment:
- [ ] As suggested by the previous  code reviewer, please kindly 
- align your button like it is in the Figma design
-  Kindly use the font color as in the Figma design as well which is a background of `#EDECFF;` and text color of `#fff`
 - Kindly add the horizontal line at the bottom as it is in the design

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/177460120-b706af42-f057-4887-89f0-3f452f46c5a2.png)  | ![image](https://user-images.githubusercontent.com/52934128/177460178-cfca786d-9b6a-46d7-afcd-6c825b247ff5.png)  |